Tree_Preorder_Traversal/Tree_Preorder_Traversal.js.js

```diff
@@ -0,0 +1,93 @@
+
+//Tree_Preorder_Traversal in javascript
+class Node
+{
+    constructor(data)
+    {
+        this.data = data;
+        this.left = null; //left child of the node
+        this.right = null; //right child of the node
+    }
+}
+class BinarySearchTree
+{
+    constructor()
+    {
+        this.root = null;
+    }
+//data inserting in tree
+insert(data)
```

Comment:
All these subsequent function should be indented 4 spaces.

Reply:
Done mam :)